content/docs/04.6-vm-management/10-create-manage-vm.md

```diff
@@ -40,30 +64,22 @@ Additionally, Virtio is a virtualization standard for network and disk device dr
 
 <WarningBox>
 
-We recommend installing ``qemu-guest-agent`` to improve VM management. We also recommend installing Virtio drivers to ensure you can use the paravirtualized hardware properly.
+We recommend installing the QEMU guest agent to display additional details in Palette Virtual Machine Orchestrator. We also recommend installing VirtIO drivers to ensure you can use the paravirtualized hardware properly. To learn how to install the guest agent and drivers, refer to the [Red Hat](https://docs.openshift.com/container-platform/4.8/virt/virtual_machines/virt-installing-qemu-guest-agent.html) reference.
```

Comment:
I'm not a big fan of linking competition, but I couldn't find a better page to link. :/
Also the steps on RedHat page only covers Windows and RedHat based OS; Debian, Ubuntu, etc., are not there.

Reply:
@cloudmaniac Removing link to Redhat doc. How can we let users know how to install QEMU guest agent without documenting third-party software?

Reply:
Per @cloudmaniac, removing the link to OpenShift doc. We'll address this as part of future VMO plans after 4.0.